common/azure_auth_test.go

```diff
@@ -249,7 +248,8 @@ func TestAzureAuth_configureWithClientSecret(t *testing.T) {
 	aa.ClientID = "a"
 	aa.ClientSecret = "b"
 	aa.TenantID = "c"
-	aa.azureManagementEndpoint = server.URL
+	// resource management endpoints end with a trailing slash in url
+	aa.azureManagementEndpoint = fmt.Sprintf("%s/", server.URL)
```

Comment:
ditto

Reply:
noticed it after it was merged. I use had more stuff there before forgot to change it to a +.